src/components/object_store/src/mem_cache.rs

```diff
@@ -81,7 +81,7 @@ impl MemCache {
             ))
         };
 
-        let inner = PartitionedMutex::try_new(
+        let inner = PartitionedMutex::try_new_bit_len(
```

Comment:
try_new_with_bit_len?

Reply:
The methods have been renamed as suggested and proper docs has been added.